src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -81,19 +81,20 @@ def test_basic_evaluation(self, pf: PFClient, runtime: str, randstr: Callable[[s
         assert run.status == RunStatus.COMPLETED
 
         eval_run = pf.run(
-            flow=f"{FLOWS_DIR}/classification_accuracy_evaluation",
+            flow=f"{FLOWS_DIR}/eval-classification-accuracy",
             data=data_path,
             run=run,
             column_mapping={"groundtruth": "${data.answer}", "prediction": "${run.outputs.category}"},
             runtime=runtime,
             name=randstr("eval_run_name"),
         )
         assert isinstance(eval_run, Run)
-        pf.runs.stream(run=eval_run.name)
+        eval_run = pf.runs.stream(run=eval_run.name)
+        assert eval_run.status == RunStatus.COMPLETED
 
         # evaluation run without data
```

Comment:
Nit: may be we can split this to another test case to make each test run quicker

Reply:
good catch, let me split it

Reply:
splitted